Source/RageCommunity.Library/Wrappers/NativeAudioWrappers.cs

```diff
@@ -6,6 +6,12 @@ namespace RageCommunity.Library.Wrappers
 {
     public static partial class NativeWrappers
     {
-        
+        /// <summary>
+        /// Varies the horn sound for a vehicle.
+        /// </summary>
+        public static void SetVehicleHornVariation(Vehicle vehicle, int variation)
```

Comment:
After asking for input in the Discord regarding the crash from this native and the GET_VEHICLE_HORN_VARIATION native, nobody was able to provide assistance.  I recommend adding an attribute as discussed yesterday before merging
```suggestion
        [ObsoleteAttribute("This method causes an Argument exception stating address cannot be zero.  Additional testing required.")]
        public static void SetVehicleHornVariation(Vehicle vehicle, int variation)
```